cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Is this being used? If not:

```suggestion
```

Reply:
Yes, use it in `Left (lstate :: Ledger ....)` somewhere below.

Reply:
```
  case decodeFull lbs of
    Right (lstate :: Ledger.LedgerState TPraosStandardCrypto) ->
```

Reply:
That was removed when I moved the decode logic to the `LocalStateQuery` module.